### ISSUE TITLE
Color Picker problem with Ws2811 12v

### ISSUE THREAD

**Issue by tfriedrich85** [djcysmic/NeopixelBusFX#14]
Hello,

I'm using this code to run some ws2812B strips but due to the voltage drop of the 5V Strips I'm trying to use the 12V ws2811 for larger distances. Its works with this ESP Easy module as well with a different behavior with the color Picker for example is this code “FFD429” – normally orange shown as more or less purple “E959FF”.
Have you any idea what’s the reason for that?

**Comment by djcysmic** [comment]
Hello,

I think that the 12V stripes have a different color order - you can change it in the source code in lines 105-110 before compiling.

**Comment by tfriedrich85** [comment]
Thanks, w'll try it!

**Comment by clumsy-stefan** [comment]
A general question to this, did you ever look into making the METHOD (eg. strip-type) dynamic configurable at runtime? I also use different types of strips attached to various nodes and always have to make special binaries for them according to the color order.

**Comment by djcysmic** [comment]
> A general question to this, did you ever look into making the METHOD (eg. strip-type) dynamic configurable at runtime? I also use different types of strips attached to various nodes and always have to make special binaries for them according to the color order.

Yes, I've tried it many times. That was my first guess with this project, but I wasn't able to compile it with a dynamic configurable method - maybe my C++ knowledge is too bad. If someone has a good idea to implement it just let me know... :)

**Comment by clumsy-stefan** [comment]
Yep, same problem here ;) I had a quick try but couldn't get it to work, also probably because lack of C++ knowledge…

I di d a bit of research and found the same question and a possible solution here https://github.com/Makuna/NeoPixelBus/issues/103 however I couldn't really get my head around it until now… Probably I'll have a deeper look into it and see if I can understand what's proposed there…

Beseides this, thanks a lot for a cool module, I use it quite a lot! regards
